js/exporter/jspdf/draw_pdf_table.js

```diff
@@ -2,7 +2,7 @@ import { extend } from '../../core/utils/extend';
 import { isDefined } from '../../core/utils/type';
 
 // this function is large and will grow
-export function drawPdfTable(doc, table) {
+export function drawPdfTable(doc, docStyles, table) {
```

Comment:
Why '*doc*Styles' ?
These styles specify how 'table' will be drawn.
Let's rename it to 'styles'